ds3-sdk/src/main/java/com/spectralogic/ds3client/models/BulkObject.java

```diff
@@ -162,7 +162,7 @@ public boolean equals(final Object obj) {
                 && nullableEquals(this.getName(), bulkObject.getName())
                 && this.getOffset() == bulkObject.getOffset()
                 && this.getPhysicalPlacement() == bulkObject.getPhysicalPlacement()
-                && this.getVersion() == bulkObject.getVersion();
+                && this.getVersionId() == bulkObject.getVersionId();
```

Comment:
This should be updated to use the object equals since the `versionId` is a `UUID` instead of a `long`

Reply:
Sigh.  Working on it, and running the tests!